Integrations/integration-FireEyeHX.yml

```diff
@@ -1752,7 +1820,7 @@ script:
             'ReadableContentsFormat': formats['markdown'],
             'HumanReadable': '## Search Results\n' + '\n'.join(md_entries),
             'EntryContext': {
-                "FireEyeHX.Search(obj._id==val._id)": results
+                "FireEyeHX.Search": search_results_to_context(results, search_id)
```

Comment:
can we pull the results again a specific search? if so maybe add (obj._id==val._id)